Makefile

```diff
@@ -1,14 +1,38 @@
-.PHONY: setup start build deploy
```

Comment:
Since I don't have any file or folder called `setup`, `start`, `build` and `deploy`, we dont need `.PHONY`. I assumed that none proccess generates this files/dirs at the end.
I add manually if something generates it. For example: `make build` created a `build` directory. So, in that case, I add it manually.
That's just how I use `.PHONY`. Feel free to use the way you think it's right. ✌️

Reply:
I think this is a nice approach, never think about it this way